rhizome/postgres/bin/configure

```diff
@@ -0,0 +1,46 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require "json"
+require_relative "../../common/lib/util"
+
+configure_hash = JSON.parse($stdin.read)
+
+# Update postgresql.conf
+configs = configure_hash["configs"].map { |k, v| "#{k} = #{v}" }.join("\n")
+safe_write_to_file("/etc/postgresql/15/main/conf.d/001-service.conf", configs)
+
+# Update pg_hba.conf
+pg_hba_entries = <<-PG_HBA
+# PostgreSQL Client Authentication Configuration File
+# ===================================================
+#
+# Refer to the "Client Authentication" section in the PostgreSQL
+# documentation for a complete description of this file.
+
+# TYPE  DATABASE        USER            ADDRESS                 METHOD
+# Database administrative login by Unix domain socket
+local   all             postgres                                peer
+
+# "local" is for Unix domain socket connections only
+local   all             all                                     peer
+# IPv4 local connections:
```

Comment:
maybe we can add the private subnet range here to allow private connections?

Reply:
Good idea, I'll ping you about this.